jit_maker/src/jit_maker.py

```diff
@@ -227,18 +229,12 @@ def user_filter(user_account, user_key: str, order) -> bool:
                     logger.info(f"best bid price: {best_bid_price}")
                     logger.info(f"best ask price: {best_ask_price}")
 
-                    start_time = time.time()
-                    await place_resting_orders(
-                        self.drift_client,
-                        perp_market_account,
-                        oracle_price_data,
-                        (best_bid_price + best_ask_price) // 2,
-                    )
-                    end_time = time.time()
-
-                    logger.info(
-                        f"resting orders placed in {start_time - end_time} seconds"
-                    )
+                    # await place_resting_orders(
+                    #     self.drift_client,
+                    #     perp_market_account,
+                    #     oracle_price_data,
+                    #     (best_bid_price + best_ask_price) // 2,
+                    # )
```

Comment:
so this wont be leaving rest orders anymore, just updating jit params letting jit proxy handle the rest?